MetaLamp/lending-pool/src/Plutus/PAB/Simulation.hs

```diff
@@ -161,32 +168,32 @@ runLendingPoolSimulation = void $ Simulator.runSimulationWith handlers $ do
         _                                                -> Nothing
     Simulator.logString @(Builtin AaveContracts) $ "Successful repay"
 
-    _ <- Simulator.callEndpointOnInstance userCid "fundsAt" sender
-    v <- flip Simulator.waitForState userCid $ \json -> case (fromJSON json :: Result (Monoid.Last (Either Text Aave.UserContractState))) of
+    _ <- Simulator.callEndpointOnInstance cidInfo "fundsAt" sender
+    v <- flip Simulator.waitForState cidInfo $ \json -> case (fromJSON json :: Result (Monoid.Last (Either Text Aave.InfoContractState))) of
             Success (Monoid.Last (Just (Right (Aave.FundsAt v)))) -> Just v
             _                                                     -> Nothing
     Simulator.logString @(Builtin AaveContracts) $ "Final user funds: " <> show v
 
-    _ <- Simulator.callEndpointOnInstance lenderCid "fundsAt" lender
-    v <- flip Simulator.waitForState lenderCid $ \json -> case (fromJSON json :: Result (Monoid.Last (Either Text Aave.UserContractState))) of
+    _ <- Simulator.callEndpointOnInstance cidInfo"fundsAt" lender
```

Comment:
For me simulation block indefinitely after Slot 74. Seems like the reason is that `userCid` and `lenderCid` are misused somewhere